lib/Mojolicious/Plugin/OpenAPI.pm

```diff
@@ -110,7 +111,8 @@ sub _before_render {
   return unless $has_spec or grep { $path =~ /^$_/ } @{$c->stash('openapi.base_paths')};
 
   my $format = $c->stash('format') || 'json';
-  my $res = $args->{exception} ? EXCEPTION() : !$has_spec ? NOT_FOUND() : NOT_IMPLEMENTED();
+  my $res = $args->{exception} ? EXCEPTION() : !$has_spec
+    ? NOT_FOUND() : $c->openapi->not_implemented($has_spec);
```

Comment:
You don't need to pass in the spec, the use can call `$c->openapi->spec` from inside the helper instead.